README.md

```diff
@@ -45,127 +56,100 @@ flowchart LR;
+3. Set up the necessary environment as described in the main [Getting Started](#-getting-started) section.
+4. Use the provided Docker scripts to run the required services.
+5. Integrate the service APIs into your application using the documentation as a guide.
 
-## 🚀 Live Docs
-
-- [Live Docs](https://projectlibertylabs.github.io/gateway/)
-
-<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+For a practical example of how these services can be used together, check out our [Social App Template](https://github.com/ProjectLibertyLabs/social-app-template).
 
 <!-- GETTING STARTED -->
 
-## 💻 Getting Started
+## 💻 Getting Started <a name="getting-started"></a>
 
-For a more detailed tutorial, visit [Live Docs](https://projectlibertylabs.github.io/gateway/).
+Gateway offers flexibility in how you can use its services. You can either quickly set up all services or selectively use specific microservices based on your needs. Below, we provide both a Quick Start guide for setting up all services and information on how to get started with individual microservices please see take a look at each service here.
 
-To get a local copy up and running, follow these steps.
+### 🚀 Quick Start Guide <a name="quick-start-guide"></a>
 
-### Prerequisites
+Follow these steps to quickly get all Gateway services up and running:
 
+1. **Prerequisites**:
 In order to run this project you need:
+  - [Docker](https://www.docker.com) or Docker compatible layer for running Gateway Services
+  - [mdBook](https://rust-lang.github.io/mdBook/) for building documentation
 
-- [Docker](https://www.docker.com) or Docker compatible layer for running Gateway Services
-- [mdBook](https://rust-lang.github.io/mdBook/) for building documentation
-
-### Setup
-
-Clone this repository to your desired folder:
-
-Clone:
-
+2. **Clone the Repository**:
 ```sh
   git clone git@github.com:ProjectLibertyLabs/gateway.git
   cd gateway
 ```
 
-### Usage
-
-To run all Gateway services, execute the following command:
+3. **Start all Services**
 
 ```sh
   ./start.sh
 ```
+This script will start all Gateway microservices using Docker.
 
-To stop the Gateway services, execute the following command:
+4. **Stop all Services**
 
 ```sh
   ./stop.sh
 ```
+Use this script when you need to stop all running Gateway services.
 
-To build the Gateway Documentation:
+5. **Build Gateway Documentation**
 
 ```sh
   cd docs
   mdbook build
 ```
 
-To build and serve the Gateway Documentation:
+6. **Build and Serve Gateway Documentation**
 
 ```sh
   cd docs
   mdbook serve
 ```
 
-### Deployment
+## Deployment <a name="deployment"></a>
 
 Deployment of the Gateway services have various options. See the [Live Docs](https://projectlibertylabs.github.io/gateway/) for more details.
 
 Deployment of the Gateway documentation occurs via merge to `main` branch via GitHub Actions.
 
-<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
```

Comment:
Do we want to get rid of these? It is a pattern and feels odd to break it just down here.

Reply:
I had removed all of them, but I can add them back. I thought the page was fairly short to scroll up. I'll add them back.